core/src/components/item/item.md.vars.scss

```diff
@@ -188,7 +188,7 @@ $item-md-label-slot-end-margin-start:       $item-md-label-slot-end-margin-end !
 // --------------------------------------------------
 
 /// @prop - Font size of a note in the start/end slot
-$item-md-note-slot-font-size:               11px !default;
+$item-md-note-slot-font-size:               0.6875rem !default;
```

Comment:
Material Design doesn't call for a minimum font size, but it is really small at 80% inside of an item. Should we give it one?